src/main/java/ch/njol/skript/expressions/ExprEntityStorageEntities.java

```diff
@@ -0,0 +1,177 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.Math2;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.Block;
+import org.bukkit.block.EntityBlockStorage;
+import org.bukkit.entity.Bee;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.function.Consumer;
+
+@Name("Entity Storage Entities")
+@Description({
+	"The entities stored inside an entity block storage (i.e. beehive).",
+	"This does not return a list of all the entities stored inside the block. This can only return the number of entities stored.",
```

Comment:
so why is it named stored entities? Shouldn't it be named stored entity count?
the ability to store an entity in a block should probably be its own effect

Reply:
Thats what I had, but Kenzie and Efy said to put into 1

Reply:
i think it's good to combine IF the expression returned entities, but since it doesn't, I think it's just going to cause confusion

Reply:
So basically revert back to an effect for adding entities, effect for clearing entities, and the expressions stays for current number  of entities and max entities

Reply:
That's what I'd prefer but idk what thoughts @Efnilite and @Moderocky  have